### PR TITLE
Unified Cloud Navigation screenshot removal and style edits 2

### PR DESCRIPTION
**Pull request by mgarrell777**
### What

This PR is meant to address style and screenshot issues related to the unified cloud navigation effort on certain pages. The pages affected by this PR are:

https://www.terraform.io/cloud-docs/run/remote-operations
https://www.terraform.io/cloud-docs/registry/add
https://www.terraform.io/cloud-docs/registry/publish-providers
https://www.terraform.io/cloud-docs/registry/publish-modules
https://www.terraform.io/cloud-docs/registry/using
https://www.terraform.io/cloud-docs/registry/design
https://www.terraform.io/cloud-docs/cost-estimation

Additional PRs will be coming to address similar screenshot and other style issue on other Terraform Cloud documentation pages.

### Why

Language style issues

Removes references to variations of a "sidebar", a "navigation bar", a "top navigation bar", and other directional language such as "upper right corner". With the UI changes, removing specific references to a navigation bar and specific directional references makes the text more agnostic.

Removes unnecessary screenshots

Screenshots are a potential hazard in documentation. Maintaining them is difficult because, as the product changes, screenshots must also change, sometimes drastically or with relative frequency. The maintenance effort places a time and resource burden on both developers and technical writers that takes away from other projects. Screenshots also present potential accessibility challenges, both for disabled users and for potential future translation for non-English speaking users.

Our stance is that screenshots are typically not needed in the documentation because text should clearly, accurately, and succinctly describe processes and results a user may encounter. Our stance is similar to the guidance in the [Google Developers Style Guide](https://developers.google.com/style/images?hl=en), which technical writing uses as a supplement to HashiCorp's own guidance.

If contributors want to include a screenshot, they must provide a rationale explaining why the screenshot provides significant value for users beyond the text. We typically use screenshots when the information in the screenshot is very difficult to explain in words. Screenshots may be helpful when the UI is non-intuitive or especially complex. They may also be useful to show the results of a complex action, like pushing metadata to the UI.

The specific screenshots to be removed have been marked in the text and will be properly removed following review.

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/registry/design.mdx [hunk 1]
```diff
@@ -53,6 +57,7 @@ Once you set a value for all of a module's required variables, its **Configure**
 Variable values can be literal strings, or can interpolate other values. When you start typing an interpolation token (`${`), the designer displays a help message. As you continue typing, it searches the available outputs in your other selected modules, as well as outputs from workspaces where you are authorized to read state outputs. You can select one of these search results, or type a full name if you need to reference a value Terraform Cloud does not know about.
 \[permissions-citation]: #intentionally-unused---keep-for-maintainers
```
Comment (laurapacilio) [inline website/docs/cloud-docs/registry/design.mdx:58]:
```suggestion
[permissions-citation]: #intentionally-unused---keep-for-maintainers
```

File: website/docs/cloud-docs/registry/publish-providers.mdx [hunk 2]
```diff
@@ -106,14 +106,15 @@ Use the Terraform Cloud API to create, read, update, and delete the following:
 
 In addition to the [Registry Providers API](/cloud-docs/api-docs/private-registry/providers#delete-a-provider), you can delete providers and provider versions through the Terraform Cloud UI. To delete providers and versions in the UI:
 
-1. Click **Registry** in the top navigation bar and click the private provider to go to its details page.
+1. Click **Registry** and click the private provider to go to its details page.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/registry/publish-providers.mdx:109]:
```suggestion
1. Click **Registry** and then click the private provider to go to its details page.
```

File: website/docs/cloud-docs/registry/publish-providers.mdx [hunk 3]
```diff
@@ -89,7 +89,7 @@ The version is available in the Terraform Cloud UI. Consumers can begin using th
 
 Consumers cannot use a private provider version until you upload all required [release files](#release-files). To determine whether these files have been uploaded:
 
-1. Click **Registry** in the top navigation bar and click the private provider to go to its details page.
+1. Click **Registry** and click the private provider to go to its details page.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/registry/publish-providers.mdx:92]:
```suggestion
1. Click **Registry** and then click the private provider to go to its details page.
```

File: website/docs/cloud-docs/registry/using.mdx [hunk 4]
```diff
@@ -14,8 +14,9 @@ All users in an organization can view the Terraform Cloud private registry and u
 
 ## Finding Providers and Modules
 
-To find available providers and modules, click the **Registry** button in the main navigation bar. The **Registry** page appears.
+To find available providers and modules, click the **Registry** button. The **Registry** page appears.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/registry/using.mdx:17]:
```suggestion
To find available providers and modules, click **Registry**. The **Registry** page appears.
```

File: website/docs/cloud-docs/run/remote-operations.mdx [hunk 5]
```diff
@@ -93,6 +93,7 @@ There are three ways to run speculative plans:
 
 If a speculative plan fails due to an external factor, you can run it again using the "Retry Run" button on its page:
```
Comment (laurapacilio) [inline website/docs/cloud-docs/run/remote-operations.mdx:94]:
```suggestion
If a speculative plan fails due to an external factor, click **Retry Run** to try it again.
```

**Comment by mgarrell777** [comment]
Closed because accidentally deleted a necessary file, then re-cloned the repo and deleted the associated local branch. Restarted in   [this PR](https://github.com/hashicorp/terraform-docs-common/pull/207)

**Pull request closed without merging**